cassandra-commons/src/main/java/com/mesosphere/dcos/cassandra/common/config/CassandraConfig.java

```diff
@@ -67,6 +69,7 @@
         private Location location;
         private int jmxPort;
         private boolean publishDiscoveryInfo;
+        private String rollingRestartName;
```

Comment:
I used Name rather than UUID, because code refers target config UUID as targetName. So wanted to keep consistency.